src/structured-data-tags.js

```diff
@@ -10,11 +10,51 @@ function ldJson(type, fields) {
   };
 }
 
-export function StructuredDataTags({structuredData = {}}, config, pageType, data, {url}) {
+function generateArticleData (story = {}, publisherConfig = {}){
+  const metaKeywords = story.seo && story.seo['meta-keywords'] || [];
+  const { authors = [] } = story;
+  return {
+    "author": authors.map(author => ({
+        "@type": "Person",
+        "givenName": author.name,
+        "name": author.name
+    })),
+    "editor": story['owner-name'],
+    "keywords": metaKeywords,
+    "publisher": publisherConfig['publisher-name'],
+    "url": `${publisherConfig['sketches-host']}/${story.slug}`,
+    "dateCreated": new Date(story['created-at']),
+    "dateModified": new Date(story['updated-at']),
+  }
+}
+
+function generateNewsArticleData (story = {}, publisherConfig = {}) {
+  return {
+    'headline' : story.headline,
+    "alternativeHeadline": story.subheadline,
+    "image": [`${publisherConfig['cdn-image']}/${story['hero-image-s3-key']}`],
```

Comment:
Are we showing the full res image? Shouldn't we compress and control the dimensions?